File: PHP PowerPoint Examples/TableApiDemo/deleteSlideTable.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\TableApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TableApi($configuration);

$name = "DeleteSlideTable.pptx"; 
$password = null; 
$folder = "Table"; 
$storage = null; 
$slideIndex = 0;
$shapeIndex = "0";
$apiInstance->deleteSlideTable($name, $slideIndex, $shapeIndex, $password, $folder, $storage);

File: Python PowerPoint Examples/TableApiDemo/deleteSlideTable.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.table_api.TableApi(configuration)

fileName = "AddSlideTable.pptx"
password = ""
storage = ""
slide_index = 0
shape_index = 0
inputFolder = "input"
result =api.delete_slide_table(fileName, slide_index, shape_index=shape_index,password=password, storage=storage, folder=inputFolder)